src/AuditTrailModule/gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
```

Comment:
Is this intentional that we are introducing gradlew files? Can't these be autogenerated as explained in https://docs.gradle.org/current/userguide/gradle_wrapper.html? 

Pardon my limited `gradle` knowledge. But in general if we can avoid having these files in the repository itself it would be better. Also, I don't see these in Community Commons module as well.

Reply:
As far as I can see in that documentation, the idea is precisely to keep the wrapper (i.e. the gradlew) file, which will then ensure you always run with the correct version and, if gradle is not available, fetch it for you.

Reply:
Aah I see. Indeed it does mention that.